examples/widgets/scale/lv_example_scale_8.c

```diff
@@ -0,0 +1,45 @@
+#include "../../lv_examples.h"
+#if LV_USE_SCALE && LV_BUILD_EXAMPLES
+
+
+/**
+ * A simple round scale with label/tick translation
+ */
+void lv_example_scale_8(void)
+{
+    lv_obj_t * scale_line = lv_scale_create(lv_screen_active());
+
+    lv_obj_set_size(scale_line, 150, 150);
+    lv_scale_set_mode(scale_line, LV_SCALE_MODE_ROUND_INNER);
+    lv_obj_set_style_bg_opa(scale_line, LV_OPA_COVER, 0);
+    lv_obj_set_style_bg_color(scale_line, lv_palette_lighten(LV_PALETTE_RED, 5), 0);
+    lv_obj_set_style_radius(scale_line, LV_RADIUS_CIRCLE, 0);
+    lv_obj_set_style_clip_corner(scale_line, true, 0);
+    lv_obj_align(scale_line, LV_ALIGN_LEFT_MID, LV_PCT(2), 0);
+    lv_obj_set_style_transform_rotation(scale_line, LV_SCALE_LABEL_ROTATE_MATCH_TICKS | LV_SCALE_LABEL_ROTATE_KEEP_UPRIGHT,
+                                        LV_PART_INDICATOR);
+    lv_obj_set_style_translate_x(scale_line, 10, LV_PART_INDICATOR);
+    lv_obj_set_style_length(scale_line, 15, LV_PART_INDICATOR);
+    lv_obj_set_style_radial_offset(scale_line, 10, LV_PART_INDICATOR);
+    lv_obj_set_style_length(scale_line, 10, LV_PART_ITEMS);
+    lv_obj_set_style_radial_offset(scale_line, 5, LV_PART_ITEMS);
+
+    lv_scale_set_label_show(scale_line, true);
+
+    lv_scale_set_total_tick_count(scale_line, 31);
+    lv_scale_set_major_tick_every(scale_line, 5);
+
+    lv_obj_set_style_length(scale_line, 5, LV_PART_ITEMS);
+    lv_obj_set_style_length(scale_line, 10, LV_PART_INDICATOR);
+    lv_scale_set_range(scale_line, 10, 40);
+
+    lv_scale_set_angle_range(scale_line, 270);
+    lv_scale_set_rotation(scale_line, 135);
+
+    lv_obj_t * needle_line = lv_line_create(scale_line);
+
+    lv_obj_set_style_line_width(needle_line, 6, LV_PART_MAIN);
+    lv_obj_set_style_line_rounded(needle_line, true, LV_PART_MAIN);
```

Comment:
```suggestion
    lv_obj_t * needle_line = lv_line_create(scale_line);

    lv_obj_set_style_line_width(needle_line, 6, LV_PART_MAIN);
    lv_obj_set_style_line_rounded(needle_line, true, LV_PART_MAIN);
    static const lv_point_precise_t points[] = {{75, 75}, {50, 50}};
    lv_line_set_points(needle_line, points, 2);
```

Did you mean to set some points so the line is visible?

Reply:
Good spot I missed lv_scale_set_line_needle_value

Reply:
The radial offset of the labels can be adjusted using PAD_RADIAL, but this is missing from the docs. I will fix